src/main/java/seedu/duke/logic/commands/Command.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke.logic.commands;
+
+import seedu.duke.logic.commands.exceptions.DeleteIndexOutOfBoundsException;
+import seedu.duke.model.ApplicationManager;
+import seedu.duke.ui.exceptions.EmptyTableException;
+
+public abstract class Command {
+    public Command() {
+    }
+    public abstract void execute(ApplicationManager applicationManager)
+            throws DeleteIndexOutOfBoundsException, EmptyTableException;
+
+    public boolean isRunning() {
+        return true;
+    }

```

Comment:
This method always returns true, so how does it track state? If this is meant to represent what this command implies (i.e. if the command the current object contains means the application continues running) then i think you should name the function something else like `shouldContinue` or `getNextRunState`